src/internal_events/udp.rs

```diff
@@ -4,6 +4,8 @@ use vector_core::internal_event::InternalEvent;
 
 use crate::{emit, internal_events::SocketOutgoingConnectionError};
 
+// TODO: Get rid of this. UDP is connectionless, so there's no "successful" connect event, only
+// successfully binding a socket that can be used for receiving.
```

Comment:
What's blocking these TODOs today?

Reply:
Nothing besides me trying to keep this PR smaller.

Reply:
That was my assumption, but wanted to double check!